src/js/player.js

```diff
@@ -1168,13 +1168,9 @@ Player.prototype.selectSource = function(sources){
  * @return {String} The current video source when getting
  * @return {String} The player when setting
  */
-Player.prototype.src = function(source){
+Player.prototype.src = function(source=this.techGet('src')){
```

Comment:
We disagree here, I don't like spaces in default args for some reason.

But I realize I'm probably being weird there, so I'll defer.